gen/dcos-config.yaml

```diff
@@ -1538,6 +1538,21 @@ package:
         ## calculation of percentiles. Raising this limit increases the accuracy
         ## of percentiles but also increases the memory usage and cpu time.
         percentile_limit = 1000
+      # Read metrics from Admin Router Nginx Prometheus endpoint.
+      [[inputs.prometheus]]
+        ## An array of urls to scrape metrics from.
+        urls = ["http://127.0.0.1:61001/nginx/metrics"]
```

Comment:
Does this port work on a strict cluster? Should we use 61002 with https instead (and setup TLS)?

Reply:
OK, I see the EE PR handles this.